src/components/ResultItem/ResultItem.tsx

```diff
@@ -1,17 +1,32 @@
 import React from 'react';
 
+type meta = {
+  type?: string;
+  date?: Date;
+  lastUpdated?: Date;
```

Comment:
I think we should make these strings for now, although I'm not actually sure what data we will pass to these going forwards.